src/Connect.php

```diff
@@ -276,15 +274,124 @@ public static function upgrade()
             $wpdb->query($sql);
             update_option('pagbank_db_version', '4.12');
         }
+
+        if (version_compare($stored_version, '4.13', '<')) {
+            $settingsTable = $wpdb->prefix . 'options';
+            $settings = get_option('woocommerce_rm-pagbank_settings');
+            $generalSettings = array();
+            $ccSettings = array();
+            $pixSettings = array();
+            $boletoSettings = array();
+            foreach ($settings as $key => $setting) {
```

Comment:
ver se o settings é um array, pois em instalações novas, ao ativar o plugin temos o erro: Warning: foreach() argument must be of type array|object, bool given in /var/www/html/wp-content/plugins/pagbank-connect/src/Connect.php on line 285